ml-agents/mlagents/trainers/demo_loader.py

```diff
@@ -61,10 +64,17 @@ def load_demonstration(file_path):
 
     # First 32 bytes of file dedicated to meta-data.
     INITIAL_POS = 33
-
-    if not os.path.isfile(file_path):
+    file_paths = []
```

Comment:
If you want to recurse into subfolders too, `os.walk()` is your friend. https://docs.python.org/3/library/os.html#os.walk

Reply:
I actually specifically didn't want to walk through all of the subfolders. This was probably biased by my own workflow where I put the .demos that I don't want to use in an "old" folder. Open to changing it if it's less confusing to the user.

Reply:
Sounds good. Not sure what's more intuitive in general.